frontend/src/slices/notifications/actions.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from '@reduxjs/toolkit';
+
+const showErrorNotification = createAction<string>('showErrorNotification');
```

Comment:
Rename it to notify, and also we should get notification service from extra and use it like here
```
 const { notification } = extra;

  notification[type](message);
```

Reply:
I apologize for any confusion, and I appreciate your efforts to explain how to implement the requested changes. To ensure I'm aligning with the intended modifications, I'd like to request a bit more context regarding the purpose of these changes. This will help me proceed in the right direction.

The comment you provided is from the file located at frontend/src/slices/notifications/actions.ts. Based on this, I assume that the proposed changes should be applied within this file. To utilize the suggested code within action.ts, it seems necessary to incorporate createAsyncThunk in order to gain access to additional functionalities.

Even though notifications may not be asynchronous now and it may be an overkill for the current implementation I don't have the full scope of error handling and what would it do for the final project. regardless of that I did it and it works but the thing is I'm not sure if this is what you wanted to change for that piece of code:

`import { createAsyncThunk } from '@reduxjs/toolkit';
import { type AsyncThunkConfig } from '#libs/types/async-thunk-config.type.js';
import { NotificationType } from '#libs/packages/services/libs/enums/notification-type.enum.js';

export const notify = createAsyncThunk<void, { message: string; type: NotificationType }, AsyncThunkConfig>(
  'notifications/notify',
  ({ message, type }, { extra }) => {
    const { notifications } = extra;
    notifications[type](message);
  }
);`

if this is not what you expect from the changes please let me know being a bit more specific on the intended change so I can have a better understanding and implement what is required from me.

Reply:
Check my PR hope gives you an idea of how to handle this.
https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/62/files#diff-313760f1b0a3c09ee85709e33b42bae2f95cfab7476f736b107b1f7065d15022

Reply:
Yes, please check pr above and let me know if you still will have a question